arm_operation/joint_value target_joint
---
string plan_result
